ci_tools/compare_coverage_to_diff.py

```diff
@@ -0,0 +1,161 @@
+""" Functions for comparing coverage output and git diff output
+"""
```

Comment:
I suggest to change the name of this file (e.g. to `coverage_analysis_tools.py`), in order to make it clear that it is just a module (like `git_evaluation_tools.py`) and not a script (like `check_new_coverage.py`).

Reply:
Fixed 792989a1